api/src/Rbac/data/migrations.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE auth_assignment
```

Comment:
it is for db storage for somebody who will just want to use the library, as I plan to extract it to some separate package